libs/ledger-live-common/src/families/ethereum/modules/send.ts

```diff
@@ -186,7 +186,11 @@ const send: ModeModule = {
   },
 
   // This is resolution config is necessary for plugins like Lido and stuff cause they use the send mode
-  getResolutionConfig: () => ({ erc20: true, externalPlugins: true }),
+  getResolutionConfig: () => ({
+    erc20: true,
+    externalPlugins: true,
+    nft: true,
```

Comment:
mmh to be rechecked but if i recall making this a default was problematic in some cases OR maybe it was just non optimal, as we will query "nft" for every single & simple eth tx

Reply:
Yes now that we reached maturity on the NFT Metadata service (which was clearly not the case a year ago), we can afford more calls on it (which was the reason behind not activating everything everywhere). 
Anyway, the unecessary calls are counter balanced by the fact that we are filtering with [the selectors now](https://github.com/LedgerHQ/ledger-live/blob/e5f62b5ffa73b2ab19d49c47c3b003ed0a88c2ab/libs/ledgerjs/packages/hw-app-eth/src/services/ledger/index.ts#L200) so we avoid calling the backend for a selector that is simply not covered by what we can clear sign.
And it has been set by default in the send mode because of how the DAppBrower + Live SDK are working (they do not provide any specific mode, so by default it ends up being a `send`...)